src/components/MultiAvatar.js

```diff
@@ -0,0 +1,77 @@
+import React, {memo} from 'react';
+import PropTypes from 'prop-types';
+import {Image, Text, View} from 'react-native';
+import styles from '../styles/styles';
+
+const propTypes = {
+    // Array of avatar URL
+    avatarImageURLs: PropTypes.arrayOf(PropTypes.string),
+
+    // Whether this option is currently in focus so we can modify its style
+    optionIsFocused: PropTypes.bool,
+};
+
+const defaultProps = {
+    avatarImageURLs: [],
+    optionIsFocused: false,
+};
+
+const MultiAvatar = props => (
```

Comment:
This should maybe have been named `MultipleAvatars` as we try to use as descriptive names as possible and not shortened or abbreviated names.